app/move/controllers/view.js

```diff
@@ -5,6 +5,7 @@ const updateSteps = require('../steps/update')
 
 const getUpdateLinks = require('./view/view.update.links')
 const getUpdateUrls = require('./view/view.update.urls')
+const setYouthMove = require('./view/view.youth-move')
```

Comment:
I originally extracted this because I thought the timeline would need to share the code - it doesn't (for now anyway).

Reply:
(Not sure why this is showing up on the tabs commit too :( )